flint.ui/src/store/index.js

```diff
@@ -1,220 +1,409 @@
+            encpFracHums: "#$0.68#$",
+            sampleDepth: "#$26.0#$",
+            sdcmRateMultDpm: "#$12.0#$",
+            sdcmRateMultRpm: "#$0.5#$",
+            sdcmRateMultBiof: "#$0.0#$",
+            sdcmRateMultBios: "#$0.76#$",
+            sdcmRateMultHums: "#$0.05#$",
+            fracPbioToBiof: "#$0.49#$",
+            fracHumsToBios: "#$0.0#$",
+            evapoOpenRatio: "#$2.5#$",
+            bToCMaxTSMDRatio: "#$3.0#$",
+          },
+        },
+      ],
+      Modules: {
+        TransactionManagerEndOfStepModule: {
+          library: "internal.flint",
+          order: 1,
+        },
+        CalendarSequencer: {
+          library: "internal.flint",
+          order: 2,
+        },
+        PlantResidueModule: {
+          enabled: true,
+          library: "moja.flint.example.rothc",
+          order: 3,
+        },
+        SoilCoverModule: {
+          enabled: true,
+          library: "moja.flint.example.rothc",
+          order: 4,
+        },
+        RothCModule: {
+          enabled: true,
+          library: "moja.flint.example.rothc",
+          order: 5,
+        },
+        ErrorScreenWriter: {
+          enabled: true,
+          library: "moja.flint.example.base",
+          order: 10,
+          settings: {
+            log_errors_max_number: 10,
+          },
+        },
+        OutputerStream: {
+          enabled: true,
+          library: "internal.flint",
+          order: 11,
+          settings: {
+            output_filename: "Example_Rothc_Point_Stock.csv",
+            output_to_screen: true,
+            output_info_header: true,
+          },
+        },
+        OutputerStreamFlux: {
+          enabled: true,
+          library: "internal.flint",
+          order: 12,
+          settings: {
+            output_filename: "Example_Rothc_Point_Flux.csv",
+            output_to_screen: false,
+            output_info_header: true,
+          },
+        },
+      },
     },
-    actions: {
-
+  },
+  mutations: {
+    // soil_func_data_orig (state) {
+    // state.Variables[8].rainfall.transform.data_orig = "cool";
+    // console.log(state);
+    // },
+    setNewConfig(state, newValue) {
+      state.Variables[8].rainfall.transform.data_orig = newValue;
     },
-    modules: {
-
-    }
-
-})
\ No newline at end of file
+    set_pointConfig_Pool_1(state, pool_1_value) {
+      var pool_1_valuex = "#$" + pool_1_value + "#$";
+      state.Point_config["Pools"][0]["Pool 1"] = pool_1_valuex;
+    },
+    set_pointConfig_Pool_2(state, pool_2_value) {
+      console.log(state.Point_config);
+      var pool_2_valuex = "#$" + pool_2_value + "#$";
+      state.Point_config["Pools"][1]["Pool 2"] = pool_2_valuex;
+    },
+    set_pointConfig_Pool_3(state, pool_3_value) {
+      var pool_3_valuex = "#$" + pool_3_value + "#$";
+      state.Point_config["Pools"][2]["Pool 3"] = pool_3_valuex;
+      console.log(state.Point_config);
+    },
+  },
+  actions: {
+    send_pointConfig() {
```

Comment:
This is cool! Two thoughts:
1. Is store/index.js going to be the final home of all our API calls, or is there a better way to structure our application? @waridrox do you know if we can make this more modular and have e.g. store/point.js, store/rothc.js
2. Let's write a custom stringify function that can be used for all JSON dispatch, following the DRY principle.

Reply:
@waridrox I guess you are already done with RothC in the store?